src/api/authFetcher.ts

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import { Axios } from '../util/axios/axios';
+import { AuthFormValues } from './../types/auth';
+
+interface AuthProps {
+  path: 'sign-in' | 'sign-up';
+  data: AuthFormValues;
+}
+
+export const authFetcher = async ({ path, data }: AuthProps) => {
+  let requestData;
+  if (path === 'sign-in') {
+    const { email, password } = data;
+    requestData = { email, password };
+  } else if (path === 'sign-up') {
+    const { email, nickName, password } = data;
+    requestData = { email, nickName, password };
+  }
+  try {
+    const res = await Axios.post(`/api/auth/${path}`, requestData);
+    return res.data;
+  } catch (error) {
+    if (axios.isAxiosError(error)) {
+      alert(error);
+    }
+  }
```

Comment:
이러면 axiosError 가 아닌 애들은 그냥 씹혀버리니까
핸들링을 안할거면 최소한 아래 정도로 그대로 에러를 뱉긴 해야돼

```suggestion
  } catch (error) {
    if (axios.isAxiosError(error)) {
      alert(error);
    } else {
      throw error;
    }
  }
```

Reply:
아하 그렇네여 감사합니다.